ynr/apps/sopn_parsing/tests/test_parse_tables.py

```diff
@@ -84,6 +89,32 @@ def test_basic_parsing(self):
             ],
         )
 
+        def test_values_contain_header_like_strings(self):
+            dataframe = json.dumps(
+                {
+                    "0": {
+                        "0": "Name of \nCandidate",
+                        "1": "BRADBURY \nAndrew John",
+                        "2": "COLLINS \nDave",
+                        "3": "HARVEY \nPeter John",
+                        "4": "JENNER \nMelanie",
+                        "5": "Name of \nCandidate",
+                    }
+                }
+            )
+            parse_tables.values_contain_header_like_strings(dataframe)
+            self.assertEqual(RawPeople.objects.count(), 1)
```

Comment:
See above comment first, but I don't think this assert is what you will want. As your `values_contain_header_like_strings` method that you are testing doesn't create a `RawPeople` object, it takes data and then either returns `True` or `False`.

So instead I'd expect the test be setup more in a way that:
- create a row like object that you expect to return True/False depending on which you are testing
- pass the row to `values_contain_header_like_strings` method and store the result in a variable e.g.
```
result = parse_tables.values_contain_header_like_strings(dataframe)
```
- Then assert against the result as being True or False
```
self.assertTrue(result)
```